lib/config.go

```diff
@@ -461,9 +470,175 @@ func InitConfig() error {
+	//if logger.GetLogger().V(logger.Warning) {
+	//	logger.GetLogger().Log(logger.Warning, "Inside LogOccConfigs()")
+	//}
+	whiteListConfigs := map[string]map[string]interface{}{
+		"BACKLOG": {
+			"backlog_pct":                  gAppConfig.BacklogPct,
+			"request_backlog_timeout":      gAppConfig.BacklogTimeoutMsec,
+			"short_backlog_timeout":        gAppConfig.ShortBacklogTimeoutMsec,
+			"saturation_recover_threshold": gOpsConfig.satRecoverThresholdMs,
+		},
+		"SHARDING": {
+			"enable_sharding":                gAppConfig.EnableSharding,
+			"use_shardmap":                   gAppConfig.UseShardMap,
+			"num_shards":                     gAppConfig.NumOfShards,
+			"shard_key_name":                 gAppConfig.ShardKeyName,
+			"max_scuttle":                    gAppConfig.MaxScuttleBuckets,
+			"scuttle_col_name":               gAppConfig.ScuttleColName,
+			"shard_key_value_type_is_string": gAppConfig.ShardKeyValueTypeIsString,
+			"enable_whitelist_test":          gAppConfig.EnableWhitelistTest,
+			"whitelist_children":             gAppConfig.NumWhitelistChildren,
+			"sharding_postfix":               gAppConfig.ShardingPostfix,
+			"sharding_cfg_reload_interval":   gAppConfig.ShardingCfgReloadInterval,
+			"hostname_prefix":                gAppConfig.HostnamePrefix,
+			"sharding_cross_keys_err":        gAppConfig.ShardingCrossKeysErr,
+			//"enable_sql_rewrite", // not found anywhere?
+			"sharding_algo":                    gAppConfig.ShardingAlgoHash,
+			"cfg_from_tns_override_num_shards": gAppConfig.CfgFromTnsOverrideNumShards,
+		},
+		"TAF": {
+			"enable_taf":                gAppConfig.EnableTAF,
+			"cfg_from_tns_override_taf": gAppConfig.CfgFromTnsOverrideTaf,
+			"testing_enable_dml_taf":    gAppConfig.TestingEnableDMLTaf,
+			"taf_timeout_ms":            gAppConfig.TAFTimeoutMs,
+			"taf_bin_duration":          gAppConfig.TAFBinDuration,
+			"taf_allow_slow_every_x":    gAppConfig.TAFAllowSlowEveryX,
+			"taf_normally_slow_count":   gAppConfig.TAFNormallySlowCount,
+		},
+		"BIND-EVICTION": {
+			"child.executable":          gAppConfig.ChildExecutable,
+			"enable_query_bind_blocker": gAppConfig.EnableQueryBindBlocker, // where should we keep it? in
+			//"enable_bind_hash_logging" FOUND FOR SOME OCCs ONLY IN occ.def
+			"bind_eviction_threshold_pct":       gAppConfig.BindEvictionThresholdPct,
+			"bind_eviction_decr_per_sec":        gAppConfig.BindEvictionDecrPerSec,
+			"bind_eviction_target_conn_pct":     gAppConfig.BindEvictionTargetConnPct,
+			"bind_eviction_max_throttle":        gAppConfig.BindEvictionMaxThrottle,
+			"bind_eviction_names":               gAppConfig.BindEvictionNames,
+			"skip_eviction_host_prefix":         gAppConfig.SkipEvictRegex,
+			"eviction_host_prefix":              gAppConfig.EvictRegex,
+			"query_bind_blocker_min_sql_prefix": gAppConfig.QueryBindBlockerMinSqlPrefix,
+			"enable_connlimit_check":            gAppConfig.EnableConnLimitCheck,
+		},
+		"SOFT-EVICTION": {
```

Comment:
Add a section for SATURATION_RECOVERY

saturation_recover_threshold = 30
saturation_recover_throttle_rate = 30

Log bind or soft eviction only if saturation_recover_throttle_rate > 0

Reply:
added a section